doc/users/next_whats_new/endless_gif.rst

```diff
@@ -0,0 +1,7 @@
+Looping GIFs with PillowWriter
```

Comment:
Something went wrong with the formatting.

Reply:
I think the file has windows file endings. I will change that.

Reply:
I think I changed the line endings?